abyss.script.py

```diff
@@ -0,0 +1,6 @@
+from abyss import abyss_plugin_t
```

Comment:
As discussed earlier in private, abyss filters can already be reloaded using the ctrl-alt-r key combo, which means filter can be edited without having to start a new instance of IDA. Also, running abyss.script.py will cause abyss.py to install certain hooks that will not be uninstalled once abyss.scriptpy is rerun (so we'll get a lot of unwanted hooks instead). I would thus like to kindly ask you to remove abyss.script.py from this pull request.

Reply:
OK will do